src/item.cpp

```diff
@@ -2475,6 +2475,11 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
     if( is_tool() && has_flag( "USE_UPS" ) ){
         ret << _( " (UPS)" );
     }
+
+    if( has_var( "NANOFAB_ITEM_ID" ) ) {
+        ret << string_format( _( " (%s)" ), find_type( get_var( "NANOFAB_ITEM_ID" ) )->nname(1) );
```

Comment:
Need comments for translator.

Reply:
My mistake that line doesn't need translation.

As its just passing  the Name string for the item contained in the template, to the template. The name string should be already translated